src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -281,24 +281,41 @@ class WP_HTML_Processor extends WP_HTML_Tag_Processor {
 	 *
 	 * ## Current HTML Support
 	 *
-	 *  - The only supported context is `<body>`, which is the default value.
 	 *  - The only supported document encoding is `UTF-8`, which is the default value.
 	 *
+	 * @todo Verify that creating a fragment in self-contained elements works.
+	 *
 	 * @since 6.4.0
 	 * @since 6.6.0 Returns `static` instead of `self` so it can create subclass instances.
+	 * @since 6.7.0 Can create fragment in any context.
 	 *
 	 * @param string $html     Input HTML fragment to process.
 	 * @param string $context  Context element for the fragment, must be default of `<body>`.
 	 * @param string $encoding Text encoding of the document; must be default of 'UTF-8'.
```

Comment:
"must be default of ..."? 
My internal autocomplete expected an "or" here. Which likely is equivalent, but I was expecting it.

And I think at least for `$context` you need to update it, since your change was about allowing other than body.

Reply:
thanks for noticing.

`must be default of UTF-8` and `must be default or UTF-8` are very different, whereas only the default value has been allowed (and the default is UTF-8). some day we might open it up to other values, but this is there to communicate intentionally that this is a UTF-8-only interface at the moment.